### PR TITLE
License the repository under AGPLv3

### PR DESCRIPTION
**Pull request by Avamander**
By using the license one can still use proprietary software to interface with the API as the data generated by it is not under the license (only this code is). Mainly this license disallows anyone to use this specific API code without stating changes and disclosing source (assuming it has been modified). The license does not disallow commercial use.

Consent of all three contributors, @frostyfrog, @HenryDelMal and @sGerli is needed. (Github's review feature could be used too to give consent)

### CODE REVIEW

**Comment by aveao** [comment]
I asked for review by sgerli and frostyfrog. @HenryDelMal needs to give consent explicitly as he's not a member, I think.

**Comment by HenryDelMal** [comment]
I agree. The license seems to be the proper.

**Review by sGerli — APPROVED** [review]

**Comment by sGerli** [comment]
Now we are just waiting for @frostyfrog 's approval.

**Review by TheTechmage — APPROVED** [review]
I don't care much what the license is, so long as it's open source, changes are pushed back into the main tree, and people can use it for whatever they desire (API wise). From your summary (and the preamble), it sounds like that's exactly the use case for this license.

**Pull request merged**
